fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerMultiTargetTest.java

```diff
@@ -675,10 +675,21 @@ void testSingleTargetCompletion() {
             context.commit();
         }
 
+        final AtomicLong counter = new AtomicLong(0);
+        final String throwMsg = "Intentionally thrown during test";
         for (long i = 0; (i + 1) * 10 < numRecords; i ++) {
-            try (OnlineIndexer indexBuilder = newIndexerBuilder(indexes.get(0), timer).build()) {
-                // Build just a small segment of the index, using the old module
-                indexBuilder.buildUnbuiltRange(Key.Evaluated.scalar(i * 10 + 5), Key.Evaluated.scalar(i * 10 + 8)).join();
+            try (OnlineIndexer indexBuilder = newIndexerBuilder(indexes.get(0), timer)
+                    .setLimit(3)
+                    .setConfigLoader(old -> {
+                        if (counter.incrementAndGet() > 1) {
+                            counter.set(0);
```

Comment:
Since this is not reused, the resetting here is unnecessary.

Reply:
✅